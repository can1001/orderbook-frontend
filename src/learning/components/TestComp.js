import React from 'react';

const TsetComp = () => (
    <div />
);

export default TsetComp;
