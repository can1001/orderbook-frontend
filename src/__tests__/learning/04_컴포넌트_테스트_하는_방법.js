import { shallow } from 'enzyme';
import React from 'react';
import TestComp from '../../learning/components/TestComp';

describe('TestComp 테스트', () => {
    it('TestComp를 렌더링하면 div여야 한다.', () => {
        const shallowWrapper = shallow(<TestComp />);
        expect(shallowWrapper.type()).toBe('div');
        expect(shallowWrapper).toMatchSnapshot();
    });
    it('테스트 케이스 정의 2', () => {

    });
});
