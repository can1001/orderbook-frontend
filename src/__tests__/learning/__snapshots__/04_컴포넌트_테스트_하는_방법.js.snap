// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TestComp 테스트 TestComp를 렌더링하면 div여야 한다. 1`] = `<div />`;
